libpod/pod.go

```diff
@@ -170,6 +174,16 @@ func (p *Pod) CPUQuota() int64 {
 	return 0
 }
 
+// Pid returns the pod process id namespace value
```

Comment:
This isn't getting the PID, but rather the path of the infra container's PID file?

It's also not really related to the PID namespace

Reply:
Yeah I was unsure here of what would be helpful here @mheon, because the namespace just contains `pod` or `host` or whatever the user passed or a ns file if specified. What would this be expected to return?

Reply:
Returning the mode of the namespace is what I would expect - we want to tell the user how the pod was configured. The PID of the pod's init container isn't that valuable.

Reply:
Similar comment here for me, I'd change the variable to indicate it's pidmode and not the pid to help make the support of this more clear months down the line.

Reply:
good idea @TomSweeneyRedHat. I will change to make it more explicit that it returns the mode passed by the user.